docs/README.md

```diff
@@ -0,0 +1,11 @@
+# Digital Records Repository Documentation
+
+![Diagram of DR2 components in AWS](/docs/images/dr2-diagram.png)
+
+Digital Records Repository is the Preservation Service for digital and digitised records within The National Archives. Digital records are stored, actively preserved, and accessed using a commercial off-the-shelf digital preservation managed service. Around this service we have built a control mechanism to facilitate ingest of and access to digital objects.
+
+We have divided our architecture into logical components, each with corresponding documentation.
+
+- [Court Document Preingest](/docs/preingest-courtDocuments.md)
+- [TDR Preingest]()

```

Comment:
This is missing a link, but I'm guessing it's because you are working on this?

Reply:
Yea, the file doesn't exist yet.